### DISCUSSION TITLE
Missing table in database after reinstallation

### DISCUSSION THREAD

**Discussion by DavidMndz** [nextcloud/recognize#1049] [Q&A] [answered]
Hello,

I am giving a try to the application after some time, I used it in the past but the CPU usage was so high that my Nextcloud instace was stuck. Now that it is expected to be solved the issue and we can limit the CPU, I am trying it again.

I reinstalled the app through the store and enable the recognition for objects for starting the tests, however, I can see this in the log and the same appears if I run the job in command line.

`An exception occurred while executing a query: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'nextcloud.oc_recognize_queue_imagenet' doesn't exist`

Does anyone know how I can recreate the tables? I think I deleted them after my previous attempt but I expected to have them re-created after the reinstallation.

Thanks!

**Comment by marcelklehr** [comment, marked as answer]
Hello :wave: 

you need to

1. uninstal
2.  remove all entries in the oc_migrations table that relate to recognize,
3. then re install

**Reply by DavidMndz** [reply]
Hello,

you rock, it now seems to be working perfectly, thanks a lot!

**Reply by robeatoz** [reply]
Thank you for the suggestion! It worked perfectly for me as well. To resolve the issue, I executed the following query:

```sql
DELETE FROM oc_migrations WHERE app LIKE 'recognize';
```
